README.md

```diff
@@ -1 +1,26 @@
-# shielded-state-sync
+# Shielded State Sync
+Synchronize shielded messages using fuzzy message detection (FMD).
+
+## Multi-key extraction
+This crate supports multi-key extraction by threhsoldizing the original [FMD2 scheme](https://eprint.iacr.org/2021/089). 
+
+Knowing up to `t < d` detection keys leaks no information about the other keys `d-t` keys. Here `t` is a corruption threshold parameter passed to the multi-key extraction.
+
+For each pair `(d,t)`, there is an associated set `P(d,t)` of valid leaked and filtering rates `(p_l,p_f)`.
+
+*  `p_l:=2^{-n}` and `n` is the number of (different) secret subkeys across any `t` detection keys, 
```

Comment:
```suggestion
* `p_l:=2^{-n}` and `n` is the number of (different) secret subkeys across any `t` detection keys, 
```

Reply:
Also, I'm confused by what you mean here and in the next point. Do you mean `n=2^t`, i.e. all subsets of t elements?

Reply:
Are do you mean `n=t`?